udata/forms/fields.py

```diff
@@ -297,7 +296,7 @@ def process_formdata(self, valuelist):
             self.data = valuelist
         elif valuelist:
             self.data = list(set([
-                x.strip().lower()
+                slugify(x, to_lower=True)
```

Comment:
In [filters.py](https://github.com/opendatateam/udata/blob/dev/udata/harvest/filters.py), slug and taglist seem not to be used in the rest of the code.

Should we not remove them rather than factorize the code ?